google/resource_compute_instance_template_test.go

```diff
@@ -309,6 +309,27 @@ func TestAccComputeInstanceTemplate_guestAccelerator(t *testing.T) {
 
 }
 
+func TestAccComputeInstanceTemplate_minCpuPlatform(t *testing.T) {
+	t.Parallel()
+
+	var instanceTemplate compute.InstanceTemplate
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeInstanceTemplateDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccComputeInstanceTemplate_minCpuPlatform(acctest.RandString(10)),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckComputeInstanceTemplateExists("google_compute_instance_template.foobar", &instanceTemplate),
+					testAccCheckComputeInstanceTemplateHasMinCpuPlatform(&instanceTemplate, "Intel Haswell"),
```

Comment:
May as well just make "Intel Haswell" a var that gets passed into the config and the check